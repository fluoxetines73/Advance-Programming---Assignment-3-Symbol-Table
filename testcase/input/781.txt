BEGIN
RPRINT
END
BEGIN
INSERT x string
ASSIGN x 'HelloWorld'
RPRINT
RPRINT
ASSIGN x '1234ABC'
END