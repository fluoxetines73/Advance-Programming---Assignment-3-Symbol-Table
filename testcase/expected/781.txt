
success
success
x//1
x//1
success